app/templates/admin/settings.hbs

```diff
@@ -1 +1,31 @@
-{{outlet}}
+<div class="ui container grid">
```

Comment:
Remove `container` for now. Let this take the full width. We'll deal with this at the end